extensions.json

```diff
@@ -1042,7 +1137,13 @@
     "repository": "https://github.com/shrey150/vscode-javafx-support"
   },
   "shuworks.vscode-table-formatter": {
-    "repository": "https://github.com/shuGH/vscode-table-formatter"
+    "repository": "https://github.com/shuGH/vscode-table-formatter",
+    "location": ".",
```

Comment:
Just FYI @manuth all extensions have the location as `.` by default. Let me know if you had any problems with not specifying this property.

Reply:
I got an error about `path` expecting a `string` but receiving `null`

I assume the problem's source is here:
https://github.com/open-vsx/publish-extensions/blob/297df0a13c62eb9a0f0d0fe037430810a99476c7/publish-extension.js#L47

Reply:
Oh, right, let me fix that real quick 😄

Reply:
Done! 😄